mifosng-android/src/main/java/com/mifos/mifosxdroid/online/groupdetails/GroupDetailsFragment.kt

```diff
@@ -100,8 +107,29 @@ class GroupDetailsFragment : MifosBaseFragment() {
         binding.btnActivateGroup.setOnClickListener {
             onClickActivateGroup()
         }
+        binding.layoutError.findViewById<Button>(R.id.btnTryAgain).setOnClickListener {
+            reloadOnError()
+        }
+    }
+
+    fun reloadOnError() {
+        sweetUIErrorHandler?.hideSweetErrorLayoutUI(binding.scrollGroupDetails, binding.layoutError)
+        viewModel.loadGroupDetailsAndAccounts(groupId)
+
     }
 
+    private fun showErrorLayout() {
+        val errorMessage = getStringMessage(R.string.failed_to_load_group_details)
```

Comment:
We already have showFetchingError method, please remove this method and move this logic in showFetchingError method.